__tests__/app.test.js

```diff
@@ -1,44 +1,125 @@
-const app = require("../app")
-const request = require("supertest")
-const db = require ("../db/connection.js")
-const seed = require("../db/seeds/seed")
-const testData = require("../db/data/test-data/index")
-const { string } = require("pg-format")
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
 
-afterAll(() => db.end())
-beforeEach(() => seed(testData))
 
+afterAll(() => db.end());
+beforeEach(() => seed(testData));
 
 describe("* GET/api/topics", () => {
-    test('404: non-existent route', () => {
-        return request(app)
-            .get('/api/topicsss')
-            .expect(404)
-            .then(({body: { msg }}) => {
-                expect(msg).toBe("path not found")
+  test("404: non-existent route", () => {
+    return request(app)
+      .get("/api/topicsss")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("path not found");
+      });
+  });
+
+  test("returns status code 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+
+  test("returns an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((topics) => {
+        expect(topics.body).toBeInstanceOf(Array);
```

Comment:
if you're testing below using a forEach loop then that already implies the data is an array - a better assertion here could be to test for the length to be what you expect based on how many entries there are in the test data